### ISSUE TITLE
NTR: Thickened glomerular basement membrane (level 4)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6274]
Thickened glomerular basement membrane (level 4)
parent:Abnormalities of the glomerular basement membrane  (NEW TERM)
def:Prominent GBM reflecting an increase in thickness (subjective estimate) @avizrosenberg @yongqunh

**Comment by pnrobinson** [comment]
This term exists, https://hpo.jax.org/app/browse/term/HP:0004722
I added the label as a synonym and revised the hierarchy.
